packages/tradershub/src/modals/Modals.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { Fragment } from 'react';
+import { JurisdictionModal } from '@/features/cfd/modals';
+import { RealAccountCreation } from '@/flows';
+
+/**
+ * @description The place to import and export all modals
+ * @returns  {React.ReactElement}
+ */
```

Comment:
- Why thou? is that like a modal manager?
- How about lazy loading modals?

Reply:
Because we just need to call it once. Rather call it in the top level. Will keep a strict eye on this so that no props are passed, and we dont repeat the same behaviour like appstore ModalManager

Regarding lazy-loading, The react-modal library only renders the modal content into the DOM when the modal is open. When the modal is closed, the content is removed from the DOM. Therefore i dont see the need to lazy load it imo

Reply:
This also solves the issue of modals being closed upon screen resizing due to different layout for responsive as for this file
https://github.com/shahzaib-deriv/deriv-app/blob/be260e75250e38217cf80542fb8c8a28f9b476ee/packages/tradershub/src/routes/TradersHubRoute/TradersHubRoute.tsx